layouts/shortcodes/supporters-list.html

```diff
@@ -53,6 +53,17 @@
     </div>
     {{ end }}
 </div>
+<div class="supporters-join-text">
+    <p><strong>Want to be featured here?</strong></p>
+    <p>
+        Send us your photo, quote, your experience or role,  
+        and any link you'd like to share (company website, blog, profile).
+    </p>
+    <p>
+        Email us at  
+        <a href="mailto:hello@collaboraoffice.com">hello@collabora.com</a>
+    </p>
+</div>
```

Comment:
Thanks Darshan! I think we can merge this. However, the spacing seems not good:

<img width="924" height="381" alt="Image" src="https://github.com/user-attachments/assets/956c607e-0d5b-435f-8d7c-f2ab410dbd73" />

Ideally we would be able to group the heading + description + Call to action visually. So,
* We lack space before in between the cards and the heading
* Make it a proper heading
* Reduce space between heading and description

Additionally that section could be more clear -> What is the action and how cool it is:
* The call to action can be a button 'Email us` (and under - bellow the button - the email address); Both with the same mailto url
* Make that stand out by
  * Either add purple/ subtle gradient background ( full wight)
  * Or Make it a card